x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/watcher/support/xcontent/XContentSource.java

```diff
@@ -90,8 +91,7 @@ public boolean isList() {
     /**
      * @return The source as a list
      */
-    public List<Object> getAsList() {
-        return (List<Object>) data();
+    public List<?> getAsList() { return ((List<?>) data());
```

Comment:
Same here, can you revert this change please?

Reply:
Reverted the changes in this file except one suppressed tag